ITU_MiniTwit/MiniTwit/migrations/0001_user_model.py

```diff
@@ -1,6 +1,44 @@
-# Generated by Django 4.1.5 on 2023-02-12 13:30
+# Generated by Django 4.1.5 on 2023-03-14 21:14
+
+
+import django.contrib.auth.models
+from django.db import connections, migrations, models
+import django.utils.timezone
+
+def copy_users(apps, schema_editor):
+    # Get a connection to the old database
+    old_db = connections['old_db']
+
+    # Execute a SELECT statement to retrieve the data from the old table
+    with old_db.cursor() as cursor:
+        cursor.execute("SELECT * FROM user")
+
+        # Iterate over the rows and insert them into the new table
+        for row in cursor.fetchall():
+            #print(row)
+            newUser = apps.get_model('MiniTwit', 'User')
+
+            # Copy the relevant data from the old table to the new table
+            new_user = newUser()
+
+            new_user.id = row[0]
+            new_user.password = row[3]
+            new_user.email = row[2]
+            new_user.username = row[1]
+            new_user.first_name = row[1].split()[0]  # first name is the first word in the username
+            if len(row[1].split()) > 1:
+                new_user.last_name = row[1].split()[1]
```

Comment:
We might potentially lose same data here if user is called for example "Maria Sofia Lopez" but I think that the majority of names does have two parts only.